client/src/components/MyGames.js

```diff
@@ -36,6 +36,10 @@ class MyGames extends React.Component {
         this.getGames();
     }
 
+    componentDidUpdate(){
```

Comment:
this sends constant requests for information and sometimes that means that other requests sent from this page don't work correctly; for example I wasn't able to log out until clicking 5 or 6 times. just call this.getGames() on using the filter

Reply:
I removed the immediate updates when changing the opponent search criteria.  I also changed it back so we stay on the same page.